src/.github/workflows/{% if ci == 'GitHub' %}test.yml{% endif %}.jinja

```diff
@@ -101,15 +101,22 @@ jobs:
           persist-credentials: false
       - name: Install addons and dependencies
         run: oca_install_addons
+      {%- if github_check_license %}
       - name: Check licenses
         run: manifestoo -d . check-licenses
+      {%- endif %}
       - name: Check development status
         run: manifestoo -d . check-dev-status --default-dev-status=Beta
       - name: Initialize test db
         run: oca_init_test_database
+        {%- if not github_enforce_dev_status_compatibility %}
+        if: always()
```

Comment:
Is it not [continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) instead ? 

Can you do the same for the license check ?

And ideally show a test run somewhere ?

Reply:
> Is it not [continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) instead ?

Apologies I thought that was only available at the job level.

> Can you do the same for the license check ?

Added

> And ideally show a test run somewhere ?

I'll set something setup now :)